src/main/java/org/folio/rest/impl/AddressValidator.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.rest.impl;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.folio.rest.jaxrs.model.Address;
+import org.folio.rest.jaxrs.model.User;
+
+class AddressValidator {
```

Comment:
Are you suggesting that the methods should be changed to static and this made a utility class?

I would prefer not to do that. I believe that static methods are mostly a workaround.

This code cannot be part of the entity in the domain model, as that code is generated.

If more of the domain was modelled in code, e.g. a domain service, I would want this to be a collaborator and thus is better as an instance rather than static (because it can be replaced in tests).